### PR TITLE
Sp23 sprint3

### PR DESCRIPTION
**Pull request by simranmakandar98**
* addition: test scripts
* fix: mission area image object URLs from correct S3 locations
* fix: reports for issue caused by removing default = 0
* improvement: implementation of data-cy for best practice

### CODE REVIEW

File: home/views.py [hunk 1]
```diff
@@ -1031,7 +1030,10 @@ def project_partner_info_admin(request):
                 for i in proj_ids:
                     cursor.execute("Select p.total_uno_students , p.total_uno_hours, p.total_k12_students, p.total_k12_hours \
                                     from projects_project p where p.id=%s", (str(i),))
+
+                    print(cursor.fetchall())
```
Comment (simranmakandar98) [inline home/views.py:1034]: This print statement will be removed in the next PR.

File: home/views.py [hunk 2]
```diff
@@ -1104,6 +1105,8 @@ def project_partner_info_admin(request):
                     for ids in sub_proj_ids:
                         cursor.execute("Select p.total_uno_students , p.total_uno_hours, p.total_k12_students, p.total_k12_hours \
                                  from projects_project p where p.id=%s", (str(ids),))
+
+                        print(cursor.fetchall())
```
Comment (simranmakandar98) [inline home/views.py:1109]: This print statement will be removed in the next PR.

File: home/views.py [hunk 3]
```diff
@@ -1314,6 +1317,8 @@ def engagement_info(request):
                     cursor.execute(
                         "Select p.total_uno_students , p.total_uno_hours from projects_project p where p.id=%s",
                         (str(i),))
+
+                    print(cursor.fetchall())
```
Comment (simranmakandar98) [inline home/views.py:1321]: This print statement will be removed in the next PR

File: partners/templates/registration/campus_partner_register_for_projects.html [hunk 4]
```diff
@@ -693,82 +687,21 @@
...
-// }
-
-
-
 
     $(document).on('click', '.add-form-row', function(e){
         e.preventDefault();
         //var validinput = validateForm();
         console.log("entering clonemore add button");
+        console.log("contactdataarray:");
```
Comment (simranmakandar98) [inline partners/templates/registration/campus_partner_register_for_projects.html:695]: This console.log statement will be removed in the next PR

File: projects/views.py [hunk 5]
```diff
@@ -993,6 +992,8 @@ def communityPrivateReport(request):
                     cursor.execute(
                         "Select p.total_uno_students , p.total_uno_hours from projects_project p where p.id=%s",
                         (str(i),))
+
+                    print(cursor.fetchall())
```
Comment (simranmakandar98) [inline projects/views.py:996]: This print statement will be removed in the next PR.

**Review by SachinPawaskarUNO — APPROVED** [review]
Looks good

**Pull request merged**